TLM/TLM/Util/VersionUtil.cs

```diff
@@ -35,10 +35,10 @@ public static class VersionUtil {
         // we could alternatively use BuildConfig.APPLICATION_VERSION because const values are evaluated at compile time.
         // but I have decided not to do this because I don't want this to happen automatically with a rebuild if
         // CS updates. these values should be changed manaually so as to force us to acknowledge that they have changed.
-        public const uint EXPECTED_GAME_VERSION_U = 197256464U;
+        public const uint EXPECTED_GAME_VERSION_U = 197256976U;
 
         // see comments for EXPECTED_GAME_VERSION_U.
-        public static Version ExpectedGameVersion => new Version(1, 15, 0, 5);
```

Comment:
was there another game update or this is just from beta to release?

Reply:
Current version is 1.15.0-f7. I think I forgot to bump it when releasing `11.7.0.1` fix. That update didn't change anything for mods, just a tiny bugfix in the game guide popups not showing.